src/events/multi-asset.upsert.service.ts

```diff
@@ -95,63 +98,66 @@ export class MaspTransactionsUpsertService {
             main: boolean;
           }>();
           for (const transaction of operation.transactions) {
-            // Masp assetName should match user grafitti
-            if (!users.has(transaction.assetName)) {
-              continue;
+            for (const multiAsset of transaction.multiAssets) {
+              // Masp assetName should match user grafitti
+              if (!users.has(multiAsset.assetName)) {
+                continue;
```

Comment:
I'd be curious how many multi asset actions we get for non-user graffitis. What do you think about adding a debug log via our service above this?

Reply:
Seems like a good idea to me!